### PR TITLE
RHBA-525: Update naming in Execution Servers screen

### PR DESCRIPTION
**Pull request by jomarko**
There are two replacemets:
- Server Template replaced with Server Configuration
- Kie Container rplaced with Deployment Unit

@karreiro would you have a look please?

### CODE REVIEW

File: kie-wb-common-screens/kie-wb-common-server-ui/kie-wb-common-server-ui-client/src/main/resources/org/kie/workbench/common/screens/server/management/client/resources/i18n/Constants.properties [hunk 1]
```diff
@@ -104,40 +104,40 @@ ProcessConfigView.KieSessionName=Kie Session Name
...
 RemoteView.RemoteInstanceRemoveError=Failed to remove the remote instance.
 RemoteView.Remove=Remove
-ServerContainerEmptyView.AddContainer=Add Container
-ServerContainerEmptyView.AddKieContainer=add a KIE Container
+ServerContainerEmptyView.AddContainer=Add Deployment Unit
+ServerContainerEmptyView.AddKieContainer=add a Deployment Unit
 ServerContainerEmptyView.Documentation=documentation
 ServerContainerEmptyView.LearnMoreMessage=You can learn more about this topic in the
-ServerContainerEmptyView.NoAssociatedContainersMessage=This Template currently has no containers. To get started,
+ServerContainerEmptyView.NoAssociatedContainersMessage=This Configuration currently has no deployment Units. To get started,
```
Comment (karreiro) [inline kie-wb-common-screens/kie-wb-common-server-ui/kie-wb-common-server-ui-client/src/main/resources/org/kie/workbench/common/screens/server/management/client/resources/i18n/Constants.properties]: I think we could replace here from `deployment Units` to `deployment units` :-)

**Comment by karreiro** [comment]
Jenkins please retest this.

File: kie-wb-common-screens/kie-wb-common-server-ui/kie-wb-common-server-ui-client/src/main/resources/org/kie/workbench/common/screens/server/management/client/resources/i18n/Constants.properties (hunk 1 above)
Comment (jomarko) [reply kie-wb-common-screens/kie-wb-common-server-ui/kie-wb-common-server-ui-client/src/main/resources/org/kie/workbench/common/screens/server/management/client/resources/i18n/Constants.properties]: Well seen, updated.

**Comment by karreiro** [comment]
Jenkins please retest this.

**Comment by jomarko** [comment]
jenkins please retest this

**Comment by karreiro** [comment]
Jenkins please retest this.

**Comment by jomarko** [comment]
@manstis this is ready for merge

**Pull request merged**